packages/odie-client/src/components/send-message-input/index.tsx

```diff
@@ -1,7 +1,12 @@
+import { HelpCenterSelect } from '@automattic/data-stores';
+import { HELP_CENTER_STORE } from '@automattic/help-center/src/stores';
 import { Spinner } from '@wordpress/components';
+import { useSelect } from '@wordpress/data';
 import { useCallback, useRef, useState } from '@wordpress/element';
 import { __ } from '@wordpress/i18n';
 import clsx from 'clsx';
+// eslint-disable-next-line no-restricted-imports
+import DropZone from 'calypso/components/drop-zone';
```

Comment:
[The one from WP](https://developer.wordpress.org/block-editor/reference-guides/components/drop-zone/) doesn't work?

Reply:
Works now!

Reply:
I was searching if we have one there but I guess I missed it 🙂